crates/extrinsics/src/contract_storage.rs

```diff
@@ -75,21 +77,26 @@ use super::{
     fetch_contract_info,
     url_to_string,
     ContractInfo,
-    DefaultConfig,
     TrieId,
 };
 
-pub struct ContractStorage<C: Config = DefaultConfig> {
-    rpc: ContractStorageRpc<C>,
+pub struct ContractStorage<C: Config, E: Environment> {
+    rpc: ContractStorageRpc<C, E>,
+    _phantom: PhantomData<fn() -> (C, E)>,
```

Comment:
Isn't this redundant because the type params are used in `ContractStorageRpc`?

Reply:
I will remove it. Thanks